server/Server.cpp

```diff
@@ -187,6 +203,21 @@ void Server::chunkHandler(Request req, int client_socket)
 	}
 }
 
+void Server::RUN(std::vector<Server> servers) {
+	for (size_t i = 0; i < servers.size(); ++i) {
+		servers[i].listenPort(3);
```

Comment:
should the backlog be part of the config file? Or maybe a constant we can adjust?

Reply:
Good question, I think it should be constant since it is to us to decide. I will add macros for it with next patch